lib/steep/subtyping/check.rb

```diff
@@ -1,6 +1,8 @@
 module Steep
   module Subtyping
     class Check
+      ABORT_LIMIT = ENV.fetch("STEEP_SUBTYPING_ABORT_LIMIT", 100).to_i
```

Comment:
The default limit is `100`, but you can override with `$STEEP_SUBTYPING_ABORT_LIMIT` environment variable.
Note that this is only for troubleshooting and not expecting to be configured by the user for daily uses.